src/entity/SquadPostModeration.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Post, PostType } from './posts';
+import { Source } from './Source';
+import { User } from './user';
+
+export enum SquadPostModerationStatus {
+  Approved = 'approved',
+  Rejected = 'rejected',
+  Pending = 'pending',
+}
+
+@Entity()
+export class SquadPostModeration {
```

Comment:
I am missing tests, queries for this entity? 

While we can merge this most of the time when we just add an entity in single PR we then in implementation find out that we forgot about something and then we have to add more migrations. 

So if possible we should merge this with a task of adding this entity in gql, adding queries and so on. Hope it makes sense?

Reply:
That's fair. I thought testing would be better paired with [this](https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-598) task, but what you are saying makes sense, so I could add a few here.

Reply:
@AmarTrebinjac yeah, you can just join the two tasks on this branch if you want, that would be perfect 👍

Reply:
@capJavert @AmarTrebinjac we should add query/mutation according to RFC to this entity for:
- get the list of pending posts of a specific squad
- approve/reject 1+ posts 

I think that this PR should contain this feature and we should raise a new one for post approval creation.  Wdyt?

Reply:
I'm a bit beyond this point unfortunately. It's gonna be this + [MI-598](https://dailydotdev.atlassian.net/browse/MI-598)